src/opnsense/mvc/app/controllers/OPNsense/Dnsmasq/forms/dialogDHCPrange.xml

```diff
@@ -27,19 +27,81 @@
         <id>range.constructor</id>
         <label>Constructor</label>
         <type>dropdown</type>
+        <style>selectpicker style_dhcpv6</style>
         <help>Interface to use to calculate the proper range, when selected, a range maybe specified as partial (e.g. ::1, ::400)</help>
```

Comment:
```suggestion
        <help>Interface to use to calculate the proper range, when selected, a range may be specified as a suffix (e.g. ::1, ::400)</help>
```

I'd rather not reinvent the wheel with "partial", the address isn't partial per-se, but can be considered a sufix (with padded leading zeroes).